tools/actions/merge.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
+
+# Requires [gh](https://cli.github.com/), [jq](https://jqlang.github.io), git, and grep. Also awk if you pass a URL.
+
+# This script can be used to "purple-merge" PRs that are supposed to land as a single commit, using the "Squash and Merge" feature of GitHub.
+# To land a PR with this tool:
+# 1. Run `git node land <pr-id-or-url> --fixupAll`
+# 2. Copy the hash of the commit at the top of the PR branch.
+# 3. Run `tools/actions/merge.sh <pr-id-or-url> <commit-hash>`.
+
+set -xe
+
+pr=$1
+commit_head=$2
+shift 2 || (echo "Expected two arguments" && exit 1)
+
+commit_queue_failed() {
+  pr=$1
+
+  echo "Failed to merge $pr"
+  cat output
+
+  rm output
```

Comment:
```suggestion
  rm -f output.json output
```

Reply:
I don't think we need the `-f`, do we?
```suggestion
  rm output.json output
```

Reply:
Realistically it doesn't matter. Formally we still do need it at least in the first error path: if `output` failed to be created for some reason, we should make robust cleanup and `exit` gracefully.